examples/with-jest/example.test.ts

```diff
@@ -1,7 +1,3 @@
-/**
- * @jest-environment node
- */
-
 it('receives a mocked response to a REST API request', async () => {
```

Comment:
The `with-jest` example is missing the MSW integration.

Reply:
Sorry, I don't understand? It's in the setup file.

Reply:
I think GitHub screwed up the diff view for me. Let me checkout this branch and look at the files proper.